File: MobyShop.Model/Models/Narudzbe.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace MobyShop.Models
{
    public class Narudzbe
    {
        public int NarudzbaId { get; set; }
        public string BrojNarudzbe { get; set; }
        public int KlijentId { get; set; }
        public string KlijentKorisnickoIme { get; set; }
        public DateTime Datum { get; set; }
        public bool Status { get; set; }
        public bool? Otkazano { get; set; }
        public decimal IznosBezPdv { get; set; }
        public decimal IznosSaPdv { get; set; }
        public int SkladisteId { get; set; }
        public string NazivSkladista { get; set; }
        public int KorisnikId { get; set; }
        public string KorisnikKorisnickoIme { get; set; }
    }
}

File: MobyShop.Mobile/lib/models/Narudzbe.dart
import 'dart:convert';

class Narudzbe{
  int? narudzbaId;
  String? brojNarudzbe;
  int? korisnikId;
  DateTime? datum;
  bool? status;
  bool? otkazano;

  Narudzbe({this.narudzbaId,this.brojNarudzbe, this.korisnikId, this.datum, this.status, this.otkazano});

  factory Narudzbe.fromJson(Map<String, dynamic> json){
    return Narudzbe(
      narudzbaId:int.parse(json["narudzbaId"].toString()),
      brojNarudzbe: json["brojNarudzbe"].toString(),
      korisnikId: json["korisnikId"],
      datum: json["datum"],
      status: json["status"],
      otkazano: json["otkazano"],
    );
  }

  Map<String, dynamic> toJson() => {
    "narudzbaId": narudzbaId,
    "brojNarudzbe": brojNarudzbe,
    "korisnikId": korisnikId,
    "datum": datum,
    "status": status,
    "otkazano": otkazano
  };
}